packages/compass-sidebar/src/components/navigation-items.tsx

```diff
@@ -92,7 +94,11 @@ const itemButtonWrapper = css({
   alignItems: 'center',
   minWidth: 0,
   paddingLeft: spacing[3],
-  paddingRight: spacing[1],
+});
+
+const signalContainerStyles = css({
+  flex: 'none',
+  fontSize: 0,
```

Comment:
What is this `fontSize` 0 used for?

Reply:
Popover is an inline element, this is convenient when sticking it near the text, in this particular case we don't want line height to affect the size of the parent container, we apply this technique in a few other places for leafygreen icons for example

Reply:
Gotcha, thanks for the explanation